tackle-test-generator-unit/src/org/konveyor/tackle/testgen/core/extender/TestSequenceExtender.java

```diff
@@ -1805,6 +1817,10 @@ private void augmentClassSequencePool() {
                 this.extSummary.classNotFoundTypes.add(cnfe.getMessage());
             } catch (IllegalArgumentException iae) {
                 logger.warning("Error creating constructor sequence for " + clsName + ": " + iae.getMessage());
+            } catch (ClassCastException  cce) { // todo - Exception should be revisited
```

Comment:
please use one catch phrase with OR between it types` (IllegalArgumentException  | ClassCastException  | ExceptionInInitializerError  e)`

Reply:
done